src/extensionVariables.ts

```diff
@@ -17,4 +18,8 @@ export namespace ext {
     export let outputChannel: IAzExtOutputChannel;
     export let openApiEditor: OpenApiEditor;
     export let selectedApiVersionDefinitionTreeItem: ApiVersionDefinitionTreeItem;
+
+    export let dataPlaneAccounts: DataPlaneAccount[];
+    export let workspaceProvider: AzExtTreeDataProvider;
+    export let workspaceItem: AzExtParentTreeItem & { dispose(): unknown; };
```

Comment:
Better naming for `workspace` related to Data Plane. `workspace` is a different concept in API Center.